lighthouse-core/lib/lh-error.js

```diff
@@ -236,6 +236,12 @@ const ERRORS = {
     lhrRuntimeError: true,
   },
 
+  // No Layout Stability trace events
+  NO_LAYOUT_JANK: {
```

Comment:
NO_LAYOUT_JANK sounds like a good thing to me 😆 

is it guaranteed there is always some jank?

Reply:
In about 10% of cases, layout instability is 0, and there will be no trace events.